docs/src/api.md

```diff
@@ -228,10 +228,15 @@ Schedulers
 Some useful schedulers are available below as part of the Agents.jl API:
 ```@docs
 Schedulers.fastest
+Schedulers.ByID
 Schedulers.by_id
```

Comment:
Notice that deprecated things should not be in the public documentation! (but should still be exported and documented)